File: app/src/main/java/ru/opalevapps/EveryGlic/ui/SplashActivity.java
package ru.opalevapps.EveryGlic.ui;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // start main Activity and finish splash
        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        SplashActivity.this.finish();
    }
}

File: app/src/main/java/ru/opalevapps/EveryGlic/ui/SplashActivity.kt
package ru.opalevapps.EveryGlic.ui

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import ru.opalevapps.EveryGlic.ui.MainActivity

@SuppressLint("CustomSplashScreen")
class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // start main Activity and finish splash
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}